cms/startup.py

```diff
@@ -8,17 +8,16 @@
 import cms.lib.xblock.runtime
 import xmodule.x_module
 from openedx.core.djangoapps.monkey_patch import django_db_models_options
-from openedx.core.djangoapps.theming.core import enable_theming
-from openedx.core.djangoapps.theming.helpers import is_comprehensive_theming_enabled
 from openedx.core.lib.django_startup import autostartup
-from openedx.core.lib.xblock_utils import xblock_local_resource_url
-from openedx.core.release import doc_version
-from startup_configurations.validate_config import validate_cms_config
 
 # Force settings to run so that the python path is modified
 
 settings.INSTALLED_APPS  # pylint: disable=pointless-statement
 
+from openedx.core.lib.xblock_utils import xblock_local_resource_url
```

Comment:
I had to move the `settings.INSTALLED_APPS` evaluation up in order for the imports in this line to succeed.

Reply:
That's concerning; any idea why it stopped working as it was?

Reply:
I'm not actually seeing that it was moved, am I missing something?

Reply:
@jmbowman It must have been something in the `enable_theming`/`is_comprehensive_theming_enabled` import that was importing/evaluating the settings - it's really the only difference. I'll look deeper.

@bmedx It's because the code above was deleted and re-added beneath - the diff is confusing.